pkg/operator/ceph/cluster/osd/spec_test.go

```diff
@@ -290,9 +290,9 @@ func TestStorageSpecConfig(t *testing.T) {
 	assert.NotNil(t, container)
 	container = job.Spec.Template.Spec.Containers[0]
 	assert.NotNil(t, container)
-	verifyEnvVar(t, container.Env, "ROOK_OSD_DATABASE_SIZE", "10", true)
-	verifyEnvVar(t, container.Env, "ROOK_OSD_WAL_SIZE", "20", true)
-	verifyEnvVar(t, container.Env, "ROOK_METADATA_DEVICE", "nvme093", true)
+	verifyEnvVar(t, container.Env, osdDatabaseSizeEnvVarName, "10", true)
```

Comment:
The unit tests sometimes use the strings instead of using the same consts as another way to check that the name doesn't change unexpectedly. I'd suggest not to worry about changing unit test consts, but it's fine to change these for now.

Reply:
Your suggestion looks good. So I'll remove the changes of test code.